src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -32,7 +32,9 @@ class HuiConditionalCard extends HTMLElement implements LovelaceCard {
     this.card = createCardElement(config.card);
     if (this.card) {
       this.appendChild(this.card);
```

Comment:
this card will break if we set config twice, it will have 2 cards attached to it. We need to clean that up.